README.md

```diff
@@ -48,8 +48,10 @@ Create nfs and data folders
 
 
 #### Generating Kubernetes manifests from the Helm Chart
+```cd ../```
 
-```helm template haiti bahmni-helm --output-dir haiti-distro  ```
+```helm template haiti bahmni-helm --output-dir haiti-distro ```
+###### ```cp custom-values.yaml.example to custom-values.yaml``` and use ```helm template -f custom-values.yaml haiti bahmni-helm --output-dir haiti-distro``` incase you need to override chart values
```

Comment:
Why this line in bold/title format? bold in markdown if `**` or `__`. But not sure we should put that in bold at all.

Reply:
"in case"